### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by TNA8** [AhmetSencan/MaskSplit-Self-supervised-Meta-learning-for-Few-shot-Semantic-Segmentation#3]
Could you provide an sample code or procedures for inference?

**Comment by sercanamac** [comment]
Hello @TNA8, we added a new inference script, please try it and let us know if anything goes wrong! According to other issues you opened, it seems like you only have 1 class and you just want to predict that class, in that case, you don't have to train the model on your own dataset. If you can just label 1 image in your dataset(manually), you can predict all the other images in your dataset by using the labeled one as support.

**Comment by TNA8** [comment]
Could you tell me why I don't have to train the model on my own dataset with only 1 class?
I already started training, is the model from that training not useful?
In that case, I should use the pretrained model on PASCAL?

**Comment by AhmetSencan** [comment]
We created a model that learns from unlabeled images and generalizes to few-shot segmentation problem on arbitrary classes. Therefore it is very much likely that our pretrained model can generalize to your class as well. That is why you might be able to directly use a pretrained model and do not have to train the model yourself. 

However, it is also very likely that, your class is not something our pretrained model is capable of predicting under few-shot segmentation setting. This depends on the training set and the nature of your class. So, I would not say that the model from your training is not useful. It is just that you might wanna try pretrained models as well to see which performs better.
